docs/adrs/0076-use-playwright.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "0076 Convert integration tests to Playwright"
+---
+
+# Convert integration tests to Playwright
+
+## Background
+
+As of the end of 2022, MilMove has been using [Cypress](https://www.cypress.io/) as its integration test framework for several years (long enough that the original decision to use Cypress instead of other technologies was not captured in an ADR). Our suite of integration tests, which has been built up by developers over the life of the project, run as a part of every build in our continuous integration process and ideally will verify that our browser and server code are working together correctly. Many user stories and epics will include making additions to our suite of integration tests as part of the acceptance criteria.
```

Comment:
**suggestion (non-blocking)**: I wasn't around so I have no idea why an ADR wasn't written, but we do have ADRs about some other early decisions, so idk if it's about how long ago we started using `Cypress`. Not a huge deal, but I personally like this wording a bit more:

```suggestion
As of the end of 2022, MilMove has been using [Cypress](https://www.cypress.io/) as its integration test framework for several years (unfortunately, the original decision to use Cypress instead of other technologies was not captured in an ADR). Our suite of integration tests, which has been built up by developers over the life of the project, run as a part of every build in our continuous integration process and ideally will verify that our browser and server code are working together correctly. Many user stories and epics will include making additions to our suite of integration tests as part of the acceptance criteria.
```